app/src/main/res/layout/sheet_link_id_with_patient.xml

```diff
@@ -22,7 +22,6 @@
     android:layout_height="wrap_content"
```

Comment:
The `linkidwithpatient_text` TextView has the text appearance set with a `<style>` tag instead of a `<textAppearance>` tag. Please change?